DataFormats/TrackReco/src/HitPattern.cc

```diff
@@ -334,6 +334,8 @@ uint16_t HitPattern::getHitPatternByAbsoluteIndex(int position) const {
   } else {
     uint8_t firstWordBits = HIT_LENGTH - secondWordBits;
     uint16_t firstWordBlock = hitPattern[secondWord - 1] >> (16 - firstWordBits);
+    if (!secondWordBits)
```

Comment:
what do you mean with ```!``` of a uint8?

Reply:
Just wanted this to be `true` if all bits were zero.
Testing locally (outside CMSSW) it seemed to work, but I guess I'm missing something?

Reply:
`!= 0` ?